src/services/AudiusBackend.js

```diff
@@ -2086,6 +2086,7 @@ class AudiusBackend {
   }
 
   static async signData() {
+    await waitForLibsInit()
```

Comment:
We should update the `Click to sign to authenticate ... ` message for this case because we're requesting discovery node.

I would either add an arg here for a custom message, or I would just make a separate version of this fn

static async signDiscoveryNodeRequest